libs/go/zmscli/policy.go

```diff
@@ -417,3 +417,97 @@ func (cli Zms) SetActivePolicyVersion(dn string, pn string, version string) (*st
 	}
 	return cli.dumpByFormat(message, cli.buildYAMLOutput)
 }
+
+func (cli Zms) DeletePolicyTags(dn string, pn, tagKey string, tagValue string) (*string, error) {
+	Policy, err := cli.Zms.GetPolicy(zms.DomainName(dn), zms.EntityName(pn))
+	if err != nil {
+		return nil, err
+	}
+	//meta := getPolicyMeta(Policy)
+
+	tagValueArr := make([]zms.TagCompoundValue, 0)
+	if Policy.Tags == nil {
+		Policy.Tags = map[zms.CompoundName]*zms.TagValueList{}
+	}
+
+	// except given tagValue, set the same tags map
+	if tagValue != "" && Policy.Tags != nil {
+		currentTagValues := Policy.Tags[zms.CompoundName(tagKey)]
+		if currentTagValues != nil {
+			for _, curTagValue := range currentTagValues.List {
+				if tagValue != string(curTagValue) {
+					tagValueArr = append(tagValueArr, curTagValue)
+				}
+			}
+		}
+	}
```

Comment:
please delete unused code - and the same as your service pull request - if code can be generic - do it